sdk/src/main/java/com/hedera/hashgraph/sdk/Client.java

```diff
@@ -283,6 +283,17 @@ public synchronized Client setNetwork(Map<String, AccountId> network) throws Int
         return network;
     }
 
+    public synchronized void pingSync(AccountId nodeAccountId) {
```

Comment:
Prefer `ping()` and `pingAll()` over explicit `Sync` suffix. Mostly because I think you should remove the `executeSync`.